package/lib/src/beamer_delegate.dart

```diff
@@ -40,7 +42,9 @@ class BeamerDelegate extends RouterDelegate<RouteInformation>
     this.updateFromParent = true,
     this.updateParent = true,
     this.clearBeamingHistoryOn = const <String>{},
+    List<BeamInterceptor>? interceptors,
   }) {
+    this.interceptors = interceptors ?? [];
```

Comment:
Can we do just `this.interceptors = const [],` in constructor and remove line 47?

Reply:
No, because then we cant add and remove from the list